src/views/Register.vue

```diff
@@ -200,12 +204,24 @@ export default {
     },
   },
   watch: {
+    /**
+     * Hàm lắng nghe sự thay đổi của email
+     * Author: DTSang(18/09)
+     */
     email: function () {
```

Comment:
Tại sao lại viết những hàm này nhỉ. Cơ chế 2way binding của Vuejs đâu sao k áp dụng

Reply:
em cũng đã thử v-model="this.account.email" xong em thấy khi em nhập liệu thì k thấy account thay đổi. Để tí em xem lại ạ = ((

Reply:
bởi vì em đang dùng sai đó. thay bằng v-model="account.email" nhé

Reply:
à vâng nhỉ =(((((((( cần thì this. đâu =(((((